pages/getting-started/quickstart.md

```diff
@@ -153,10 +153,25 @@ plural bundle install console console-azure
 {% /tab %}
 {% /tabs %}
 
+As of CLI version 0.6.19, the bundle name can be inferred from primary bundles, shortening the command to:
+
+```
+plural bundle install console
+```
+
 After running the install command, you will be asked a few questions about how your app will be configured, including whether you want to enable **Plural OIDC** (single sign-on). Unless you don't wish to use Plural as an identity provider due to internal company security requirements, you should enter (Y). This will enable you to use your existing `app.plural.sh` login information to access Plural-deployed applications. This will add an extra layer of security for applications without built-in authentication.
 
 Ultimately all the values you input at this step will be stored in a file called `context.yaml` at the root of your repo.
 
+Alternatively, you can install Plural bundles using our interactive GUI. To start the GUI, run:
```

Comment:
should we move this to the top?

Reply:
Can do! Will leave the rest as-is since `plural repos list` still shows bundle-specific repo names.

Reply:
moved up 👍